webapp/store/selectors/plugins.js

```diff
@@ -50,7 +48,8 @@ export const sortPluginMetadata = pluginMeta => {
 export const uniquePathNames = metadata => {
   const plugins = sortPluginMetadata(metadata);
   const paths = new Set();
-  return plugins.map(plugin => {
+  return plugins.map(_plugin => {
```

Comment:
Map vs Reduce

map feels like it should be a more stateless thing that applies a function to each element in a list where reduce maintains state and keeps it in context while iterating over each element in a list